Unix/installbuilder/datafiles/Linux.data

```diff
@@ -174,6 +174,34 @@ ConfigureOmiService() {
 %Preinstall_10
 #include OmiService_funcs
 
+# Verify that the proper version of SSL is installed (if we're on ULINUX)
+
+SSL_VERSION=`openssl version | awk '{print $2}'`
+case "$SSL_VERSION" in
+    0.9.8*)
+	SSL_FOUND=0.9.8
+        ;;
+    1.0.*)
+        SSL_FOUND=1.0.0
+        ;;
+    1.1.*)
+        SSL_FOUND=1.1.0
+        ;;
+    *)
+        echo "Preinstall script error: Unrecognized version of SSL on the system: $SSL_VERSION" >&2
+        exit 2
+        ;;
+esac
+
+if [ ${{PFDISTRO}} = ULINUX -a $SSL_FOUND != "${{SSL_BUILD_VERSION}}" ]; then
```

Comment:
As a habit, I tend to always put quotes around all string comparison operands, just to be safe.  Like  if [ "${{PFDISTRO}}" = "ULINUX" -a "$SSL_FOUND" != "${{SSL_BUILD_VERSION}}" ];

Reply:
Yeah, I thought about that. In this case, PFDISTRO will always be defined, and SSL_FOUND is defined just above. But SSL_BUILD_VERSION may not be defined (specifically if we're not on ULINUX), causing a syntax error.

In this specific case, I think this 'if' statement is fine. But I think your tendency is a fine one, and I'll make that change prior to checkin.